### PR TITLE
Feature/add contact form

### PR DESCRIPTION
**Pull request by Markotweyo**
- Added contact form
- styled contact form
- Fixed linters errors
- Pair programmed in fixing styles
# After first review
- Fixed the button hover style
- Updated formspree form_id
# After second review
- Fix button hover color and background
- Fix button active color

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -217,39 +226,62 @@ <h2 class="socialmedia">LET'S CONNECT</h2>
...
                 </ul>
             </div>
         </section>
+        <section class="contact-me">
+            <footer class="contact-me-container">
+                <h2 class="contactme-header">Contact me</h2>
+                <p class="">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <form action="#" id="contact-form" class="contact-form" method="post">
```
Comment (Whoistolu) [inline index.html:267]: - [ ] [REQUIRED] Kindly note that the form is required to use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). Ensure that the `action` value is similar to `https://formspree.io/f/{form_id}`

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Markotweyo 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -444,3 +444,8 @@ body {
   cursor: pointer;
   border: none;
 }
+
+.get-in-touch:hover {
+  background: #0d0f5b;
+  color: #fff;
+}
```
Comment (omarsalem7) [inline styles.css:451]: - [ ] As stated by the previous reviewer, please add pressed interaction when clicking on the button like this:
```
.get-in-touch:active {
  color: #2230D2;
}
```
- [ ] Also consider using correct color code for hover interaction (`#6070FF`)

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Markotweyo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull requrst

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @Markotweyo and @ifzyy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**